python/google/protobuf/json_format.py

```diff
@@ -292,7 +292,7 @@ def _FieldToJsonObject(self, field, value):
     elif field.cpp_type == descriptor.FieldDescriptor.CPPTYPE_BOOL:
       return bool(value)
     elif field.cpp_type in _INT64_TYPES:
-      return str(value)
```

Comment:
https://developers.google.com/protocol-buffers/docs/proto3?hl=zh-CN#json

From our json mapping, int63 is present by string in Josn

Reply:
I see. Is the idea that some languages don't support 64bit numbers but all support strings?

Reply:
I see. I guess JSON doesn't support 64 bit ints?